### DISCUSSION TITLE
Find Search UI Template

### DISCUSSION THREAD

**Discussion by tobyyu007** [ResilientApp/Res-a-Me#6] [Ideas]
Search UI need to have these features
- Search Bar
- Search Result

**Comment by tobyyu007** [comment]
## [vue-search-ui-demo ](https://github.com/elastic/vue-search-ui-demo/tree/master)
[Demo Website](https://elastic.github.io/vue-search-ui-demo/?size=n_20_n)
<img width="1917" alt="image" src="https://github.com/tobyyu007/Res-a-Me/assets/39978173/ceb7dafb-6d4a-4199-8669-092a20ad9449">

**Comment by tobyyu007** [comment]
## [E-commerce demo](https://github.com/algolia/instantsearch/tree/master/examples/vue/e-commerce)
[Demo Website](https://instantsearchjs.netlify.app/examples/vue/e-commerce/)
<img width="1917" alt="image" src="https://github.com/tobyyu007/Res-a-Me/assets/39978173/1fde0874-5bb6-448f-9eec-acaf7a61cf23">

**Comment by tsungchiehchen** [comment]
- https://github.com/iamstevendao/vue-suggestion
- https://github.com/fareez-ahamed/autocomplete-vuejs
    - Runnable
    - [Medium Explanation](https://medium.com/@fareez_ahamed/make-your-own-autocomplete-component-quickly-using-vue-js-21a642e8b140)
- https://primevue.org/inputmask/
    - Seems to have multiple components that can be used directly
    - Has Input box that can act as signup or login text input box
    - Has autocomplete search box

**Comment by yingchengchen** [comment]
- https://codepen.io/AndrewThian/pen/QdeOVa
    -  showing search result without redirecting to another page (maybe we can try if it is not too hard) 
- https://ej2.syncfusion.com/vue/demos/#/material/auto-complete/template.html
    -  looks like what we need!

**Reply by tsungchiehchen** [reply]
[This is awesome](https://ej2.syncfusion.com/vue/demos/#/material/auto-complete/template.html)

**Reply by tobyyu007** [reply]
如果我們用這個的話 [https://ej2.syncfusion.com/vue/demos/#/material/auto-complete/template.html]()，可能還需要找一個排版介面，還是要直接手刻？

**Comment by tobyyu007** [comment]
# Nov 10 Discussion for Search UI

## Use [Vuetify](https://vuetifyjs.com/en/components/forms/#rules)
- [Autocomplete](https://vuetifyjs.com/en/components/autocompletes/)
